### DISCUSSION TITLE
Welcome to vscode-browser-preview Discussions!

### DISCUSSION THREAD

**Discussion by auchenberg** [auchenberg/vscode-browser-preview#169] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EpochBoy** [comment]
Hey @auchenberg 

I really appreciate it that you've made this extension, I'm happily using this on the side 👍 

I'm just wondering if there is a way to tweak the visual fidelity(increase), so it could be used 💯% of the time?

Keep trucking! 💪

**Comment by Krishankantmani3** [comment]
Hey, first of all, thanks for making such a useful extension. 

let me explain briefly the way I am using this extension in my project.
1. created a docker container of Debian base image and then installed code-server and other required packages.
2.  now also installing chrome and this extension for browser-preview.

Problem: the problem is installing chrome in docker container/image increasing the huge size of docker.

Q1. is there any way to package all executable files into one folder and mount it to the docker container so that we can set the path of chrome installation in the extension settings?

Q2. if it is possible then how can we package executables files in a folder and which file do I ​have to set?

I know, this problem arises due to the container things and Linux but i am a beginner so asking here.

**Comment by LHF0608** [comment]
嘿，首先，感谢您进行如此有用的扩展。

让我简要解释一下我在项目中使用此扩展的方式。

创建了一个 Debian 基础镜像的 docker 容器，然后安装了代码服务器和其他所需的软件包。
现在还安装了Chrome和此扩展程序以进行浏览器预览。
问题：问题是在 docker 容器/映像中安装 chrome，增加了 docker 的巨大尺寸。

问题 1.有没有办法将所有可执行文件打包到一个文件夹中并将其挂载到 docker 容器中，以便我们可以在扩展设置中设置 Chrome 安装路径？

问题 2.如果可能，那么我们如何将可执行文件打包到一个文件夹中，我必须设置哪个文件？

我知道，这个问题是由于容器和 Linux 而出现的，但我是初学者，所以在这里问。
